packages/cli/src/workflow-runner.ts

```diff
@@ -401,7 +404,7 @@ export class WorkflowRunner {
 
 			// Normally also workflow should be supplied here but as it only used for sending
 			// data to editor-UI is not needed.
-			await hooks.executeHookFunctions('workflowExecuteBefore', []);
+			await hooks.executeHookFunctions('workflowExecuteBefore', [undefined, data.executionData]);
```

Comment:
```suggestion
			await hooks.executeHookFunctions('workflowExecuteBefore', [workflow, data.executionData]);
```

Reply:
I don't think many of us have tested hooks in queue mode. If you are seeing the hook being executed twice, then we should create a follow up ticket.